black.py

```diff
@@ -2161,6 +2106,22 @@ def make_comment(content: str) -> str:
     return "#" + content
 
 
+def _line_has_trailing_comma(l: Line) -> bool:
```

Comment:
Nit: the current API would put "has_trailing_comma()" as a method on Line.

Reply:
This turned into `is_collection_with_optional_trailing_comma` as a property on Line.

(Standalone tests for `is_collection_with_optional_trailing_comma` should be coming soon, but I wanted to get feedback from Travis on the rest of the CI status...)